app/views/compare/tab_content/_groups.html.erb

```diff
@@ -1 +1,14 @@
-Groups
\ No newline at end of file
+<div class="tab-pane fade show" id="groups" role="tabpanel" aria-labelledby="groups-tab">
+  <h4 class='my-2'><%= t('compare.index.groups.title') %></h4>
+
+  <div class="bg-light p-3 rounded border">
+    <%= form_tag benchmarks_compare_index_path, method: :get do %>
+      <%= hidden_field_tag(:type, 'groups') %>
+      <%= select_tag('school_group_ids', options_from_collection_for_select(SchoolGroup.all.select(&:has_visible_schools?), "id", "name", @filter[:school_group_ids]), multiple: true) %>
```

Comment:
Can you sort the groups by name? 

And also add a `size` attribute to increase the form field height to make it easier to find groups?